pkgs/development/web/bun/default.nix

```diff
@@ -31,6 +32,19 @@ stdenvNoCC.mkDerivation rec {
 
     runHook postInstall
   '';
+
+  postPhases = [ "postPatchelf" ];
+  postPatchelf = lib.optionalString (stdenvNoCC.buildPlatform.canExecute stdenvNoCC.hostPlatform) ''
+    completions_dir=$(mktemp -d)
+
+    SHELL="zsh" $out/bin/bun completions $completions_dir
+    SHELL="fish" $out/bin/bun completions $completions_dir
+
+    installShellCompletion --name bun \
+      --zsh $completions_dir/_bun \
+      --fish $completions_dir/bun.fish
```

Comment:
bun also has [bash completions](https://github.com/oven-sh/bun/blob/main/completions/bun.bash), can you add those too please?